tests/Functional/BeaumontConsultingTest.php

```diff
@@ -3,55 +3,33 @@
 namespace Tests\Functional;
 
 use EthicalJobs\Sanitize\PurifierFactory;
+use Tests\Fixtures\LoadsFixtures;
 use Tests\TestCase;
 
 class BeaumontConsultingTest extends TestCase
 {
-    /**
-     * @test
-     */
-    public function it_correctly_purifies_fixture_1()
-    {
-        $before = file_get_contents(__DIR__ . '/../Fixtures/BeaumontConsulting/fixture.1.before.html');
-
-        $after = file_get_contents(__DIR__ . '/../Fixtures/BeaumontConsulting/fixture.1.after.html');
-
-        $purifier = PurifierFactory::create();
-
-        $purified = $purifier->purify($before);
-
-        $this->assertEquals($purified, $after);
-    }
+    use LoadsFixtures;
 
     /**
-     * @test
+     * Data for it_correctly_purifies test
      */
-    public function it_correctly_purifies_fixture_2()
+    public final function data(): iterable
     {
-        $before = file_get_contents(__DIR__ . '/../Fixtures/BeaumontConsulting/fixture.2.before.html');
-
-        $after = file_get_contents(__DIR__ . '/../Fixtures/BeaumontConsulting/fixture.2.after.html');
-
-        $purifier = PurifierFactory::create();
-
-        $purified = $purifier->purify($before);
-
-        $this->assertEquals($purified, $after);
+        return $this->fixtureData('BeaumontConsulting', 3);
     }
 
     /**
      * @test
+     * @dataProvider data
+     * @param $before
+     * @param $after
      */
-    public function it_correctly_purifies_fixture_3()
+    final public function it_correctly_purifies(string $before, string $after) :void
```

Comment:
Space should come after colon, not before

Reply:
FIxed